modules/xmpp/ChatRoom.js

```diff
@@ -534,14 +534,17 @@ export default class ChatRoom extends Listenable {
         // make sure we catch all errors coming from any handler
         // otherwise we can remove the presence handler from strophe
         try {
-            let tagHandler = this.presHandlers[node.tagName];
+            let tagHandlers = this.presHandlers[node.tagName];
 
             if (node.tagName.startsWith('jitsi_participant_')) {
-                tagHandler = this.participantPropertyListener;
+                // NOTE should we allow more than one handler ?
```

Comment:
The question is whether or not  "jitsi_participant_ events" require more handlers just like other events, but it's accessed via "set" style method, so I'm leaving it as is. Should I remove the comment ?